test/parser/KotlinMetrics.test.ts

```diff
@@ -0,0 +1,170 @@
+import { testFileMetric } from "./TestHelper";
+import { FileMetric } from "../../src/parser/metrics/Metric";
+
+describe("Kotlin metric tests", () => {
+    const kotlinTestResourcesPath = "./resources/kotlin/";
+
+    describe("parses Kotlin complexity metric", () => {
+        it("should count loops properly", async () => {
+            await testFileMetric(kotlinTestResourcesPath + "loops.kt", FileMetric.complexity, 5);
+        });
+
+        it("should count if statements correctly", async () => {
+            await testFileMetric(
+                kotlinTestResourcesPath + "if-statements.kt",
+                FileMetric.complexity,
+                10
+            );
+        });
+
+        it("should not count any class declaration", async () => {
+            await testFileMetric(kotlinTestResourcesPath + "classes.kt", FileMetric.complexity, 4);
+        });
+
+        it("should count case but no default statements correctly", async () => {
+            await testFileMetric(
+                kotlinTestResourcesPath + "case-statements.kt",
+                FileMetric.complexity,
+                5
+            );
+        });
+
+        it("should count functions and methods correctly", async () => {
```

Comment:
Looks like init block is counted for complexity and functions metric for Kotlin, but not Java. Maybe we should mention this in our README.md? @ResistantBear @michaelalper-cezik-mw 
```suggestion
        it("should count all function declarations and one init block correctly", async () => {
```

Reply:
I guess the init-block is a block where you can throw arbitrary statements in, just like a function body? In this case, we should count it as a function, just like contructors, etc.

Reply:
I agree that this is like a function and should be counted accordingly for complexity and functions metric.
Do we handle init blocks for java?

Reply:
@ce-bo actually, init block are not counted in Java. But we will change that in #108